packages/website/builder/css/main.css

```diff
@@ -101,3 +101,20 @@ pre > code {
   font-size: 14px;
   line-height: 24px;
 }
+
+@font-face {
+    font-family: 'cerebral icons';
+    src: url('/images/cerebral-icons.woff') format('woff');
```

Comment:
Is woff format enough for all supported browsers?

Reply:
[Yes.](http://caniuse.com/#feat=woff)